rotkehlchen/db/evmtx.py

```diff
@@ -154,6 +155,72 @@ def get_evm_internal_transactions(
 
         return transactions
 
+    def assert_tx_data_is_pulled(
+            self,
+            cursor: 'DBCursor',
+            evm_inquirer: 'EvmNodeInquirer',
+            tx_hash: 'EVMTxHash',
+            relevant_address: Optional['ChecksumEvmAddress'],
+    ) -> tuple[tuple[Any, ...], 'EvmTxReceipt']:
+        """Makes sure that the required data for the transaction are in the database.
+        If not, pulls them and stores them. For most chains this is the transaction and the
+        receipt. Can be extended by subclasses for chain-specific information.
+
+        May raise:
+        - RemoteError if there is a problem querying the data sources or transaction hash does
+        not exist.
+        """
+        query, bindings = EvmTransactionsFilterQuery.make(tx_hash=tx_hash, chain_id=evm_inquirer.chain_id).prepare()  # noqa: E501
+        query, bindings = self._form_evm_transaction_dbquery(query=query, bindings=bindings, has_premium=True)  # noqa: E501
+        tx_data = cursor.execute(query, bindings).fetchone()
```

Comment:
kwargs in all of these functions. Please do not make me repeat things across multiple reviews